src/Application/Regulation/Command/CreateRegulationOrderHistoryCommandHandler.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\Regulation\Command;
+
+use App\Application\DateUtilsInterface;
+use App\Application\IdFactoryInterface;
+use App\Domain\Regulation\RegulationOrderHistory;
+use App\Domain\Regulation\Repository\RegulationOrderHistoryRepositoryInterface;
+
+final class CreateRegulationOrderHistoryCommandHandler
+{
+    public function __construct(
+        private IdFactoryInterface $idFactory,
+        private RegulationOrderHistoryRepositoryInterface $regulationOrderHistoryRepository,
+        private DateUtilsInterface $dateUtils,
+    ) {
+    }
+
+    public function __invoke(CreateRegulationOrderHistoryCommand $command): RegulationOrderHistory
+    {
+        $regulationOrderHistory = $this->regulationOrderHistoryRepository->add(
+            new RegulationOrderHistory(
+                uuid: $this->idFactory->make(),
+                regulationOrderUuid: $command->regulationOrder->getUuid(),
+                userUuid: $command->user->getUuid(),
```

Comment:
Je me demande si, ici, ce ne serait pas mieux d'utiliser le service `AuthenticatedUser` pour ne pas devoir passer l'information à la commande à chaque fois. Qu'en penses tu ?

Ça ferait quelque chose du genre `userUuid: $this->authenticatedUser->getUser()->getUuid()`;

Reply:
@mmarchois Ça ferait utiliser un composant Infrastructure spécifique Symfony dans la couche Application, et mettre de l'implicite dans la commande... Comme le `$user` correspond au "by" (created by, etc), ça me semble logique de l'avoir explicitement en paramètre de la commande. Donc le `$command-user` me semble préférable

Reply:
... Mais je comprends aussi l'intérêt de ne pas avoir à mettre de `User` en paramètre d'une commande où à un moment dans celle-ci on veut créer un RegulationOrderHistory

Peut-être alors juste introduire un `src/Application/AuthenticatedUserInterface.php` ? Dans une prochaine PR OK, pas bloquant pour celle-ci je dirais

Reply:
Humm, je me retrouve avec le même problème sur https://github.com/MTES-MCT/dialog/pull/1160 depuis le rebase sur main